### ISSUE TITLE
In the default setting,will the validation set be added to the training set?

### ISSUE THREAD

**Issue by cqnkjsx** [open-mmlab/mmediting#126]
Without changing the main settings,I trained my EDVR model and had a testing,but the result was strange:on the four test videos the average psnr of results were over 32,outperformed the results in the paper.

**Comment by cqnkjsx** [comment]
I used the bicubic 4x data to do the SR task

**Comment by xinntao** [comment]
Not clear about your setting. PSNR over 32, emmm, there must be some bugs~

**Comment by cqnkjsx** [comment]
> Not clear about your setting. PSNR over 32, emmm, there must be some bugs~

In fact the eval-PSNR is about 9 according to the output,but I think it is too small and change the way to calculate it
